artifacts/CAMARA_common.yaml

```diff
@@ -59,14 +59,11 @@ components:
         * `ipv4Address`
         * `ipv6Address`
         * `phoneNumber`
-        * `networkAccessIdentifier`

```

Comment:
We do not think that it is reasonable to remove the networkAccessIdentifier, even though it is not widely used yet.
We actually would like to add a further option to allow identification via the operator token.
For security reasons we see a mandatory need to allow verification of the data subject named in the access token against the target resource, which is input to the api call. We need to keep all option open, especially since we are discussing (in issue #145 in ICM group requesting to support the operator token).

Reply:
**We are focusing on a proposal in the scope of the next meta release**. The proposal to remove the networkAccessIdentifier is precisely because the feedback so far has been favourable to doing so, and there is no explicit feedback from any WG participant that they are currently using it. Of course, it could be revisited for future releases if the WG needs it, and it could be re-evaluated whether or not to include it again.

The operatorToken discussed in [ICM [#145](https://github.com/camaraproject/Commonalities/pull/145)](https://github.com/camaraproject/IdentityAndConsentManagement/issues/145) is just an open discussion for now (in ICM backlog), so I would suggest not even considering it for this meta release. In fact, operatorToken might fit as networkAccessIdentifier from our point of view, but it should be discussed for a release when there is a clear position and solution defined in CAMARA for operatorToken.

Reply:
Can we try to find a middle way here? I would propose that we could write in the guidelines that the networkAccessIdentifier is only part of the schema to be able to be future-proof and Camara currently does not permit its use. 
Post meta-release work, and after the resolution of the relevant issues in ICM, its use will have to be explicitly documented in ICM and in the guidelines. 

This will be similar to what we have done for subscriptions where even though event type is an array (for being future-proof), we are allowing for the meta release the use of just single event type and after the meta-release we will target usecases which need subscriptions to support multiple event types.

WDYT?

Reply:
If it works and helps us reach an agreement, that would be fine with me. But I think it would be better to just have an artifact definition of what needs to be supported and allowed to be used. And then just evolve it in the next releases.

Reply:
@Elisabeth-Ericsson : Could you please confirm if this suggested middle way would be ok for you as well?

Reply:
I'm fine with the middle way.

Reply:
> This will be similar to what we have done for subscriptions where even though event type is an array (for being future-proof), we are allowing for the meta release the use of just single event type and after the meta-release we will target usecases which need subscriptions to support multiple event types.

But in this case, the WG is also changing the schema in https://github.com/camaraproject/Commonalities/pull/236 to add minItems and maxItems set to "1". So with this approach, we would actually have a schema according to what will be supported in the next meta-release. Doing the analogy with our case in this PR, it would be removing NAI from the device object definition, so the artifact definition of what needs to be supported and allowed to be used.

I could try to suggest some changes in the PR to address this "middle way" in any case...

Reply:
Middle way agreement applied in https://github.com/camaraproject/Commonalities/pull/233/commits/4ae03bb1d76be80243badfdce988a8b254df0b76
Please have a look. shilpa-padgaonkar @Elisabeth-Ericsson and suggest any necessary changes to the wording I use.

Reply:
If an API client only provides the networkAccessIdentifier, we can return a 422 error, and if the networkAccessIdentifier is provided along with other allowed identifiers, it should be ignored.

Should we also add this to the NOTE included in the common.yaml file?

Reply:
This clarification is very useful, could you confirm if we can add this in our API subprojects?

Reply:
> This clarification is very useful, could you confirm if we can add this in our API subprojects?

There is general guideline in API Design Guidelines section [6.2 Error Responses - Device Object](https://github.com/camaraproject/Commonalities/blob/main/documentation/API-design-guidelines.md#62-error-responses---device-object).
The clarification above is in line with it, so it can be added in subprojects.
If more detailed guideline is needed for networkAccessIdentifier it can be added as a fix for API Design Guidelines.